### PR TITLE
Feature flags for document registration number.

### PR DESCRIPTION
**Pull request by WalterMoar**
Re-purposed one of the PoC feature flags for the Document Registration Number search. Removed broken tests until we determine what we want to unit test for the flags.

### CODE REVIEW

File: ppr-ui/src/flags/ld-client.ts [hunk 1]
```diff
@@ -27,10 +27,10 @@ class VueLdClient {
     }
   }
 
-  private updateFlags(allFlags): void {
+  private updateFlags(allFlags: LDFlagSet): void {
     const flags = FeatureFlags.Instance
     console.debug('LDFlags on change flags', allFlags)
-    flags.feature1 = allFlags['poc-feature-1']
+    flags.feature1 = allFlags['search-document-registration-number']
```
Comment (jguertin) [inline ppr-ui/src/flags/ld-client.ts:33]: Will you also rename `feature1` to reflect to the intent?

**Review by jguertin — APPROVED** [review]
looks good, 1 question, non-blocking

File: ppr-ui/src/flags/ld-client.ts (hunk 1 above)
Comment (WalterMoar) [reply ppr-ui/src/flags/ld-client.ts:33]: Yes indeed! We also need to something more scalable for the flags, but might not get to that for this sprint.

File: ppr-ui/src/views/Home.vue [hunk 2]
```diff
@@ -82,25 +70,28 @@
...
+
+    const userIsAuthed = computed((): boolean => !!sessionStorage.getItem('KEYCLOAK_TOKEN'))
+    const userCanSearch = computed((): boolean => flags.feature1 && userIsAuthed.value)
+
     function goSearch(): void {
       router.push('search')
     }
 
-    return { flags, feature2, featureOneLabel, featureTwoLabel, goSearch, userIsAuthed }
+    return { flags, featureOneLabel, featureTwoLabel, goSearch, userCanSearch, userIsAuthed }
```
Comment (bryan-gilbert) [inline ppr-ui/src/views/Home.vue]: Can also remove the two featureXXXLabel computes.

**Review by bryan-gilbert — APPROVED** [review]
one non-blocking change suggested

File: ppr-ui/src/views/Home.vue (hunk 2 above)
Comment (WalterMoar) [reply ppr-ui/src/views/Home.vue]: Yes! I completely missed that.

File: ppr-ui/src/flags/ld-client.ts (hunk 1 above)
Comment (WalterMoar) [reply ppr-ui/src/flags/ld-client.ts:33]: https://github.com/bcgov/ppr/issues/369

**Pull request merged**